docs/reference/commandline/service_create.md

```diff
@@ -1120,6 +1121,59 @@ $ docker service create --name cuda \
                         nvidia/cuda
 ```
 
+### Running as a job
+
+Jobs are a special kind of service designed to run an operation to completion
+and then stop, as opposed to running long-running daemons. When a Task
+belonging to a job exits successfully (return value 0), the Task is marked as
```

Comment:
See my comment; do we want failed tasks to be started / replaced / tried again by default? Or should it have `--restart-condition=none` ?